### ISSUE TITLE
Wrong position of filter button in Edge. 

### ISSUE THREAD

**Issue by MilenaCh** [telerik/kendo-theme-default#580]
### Bug report

Open the demo below with Default v2 or Bootstrap v4 themes in Edge:
http://demos.telerik.com/kendo-ui/grid/local-data-binding

Result:
https://www.screencast.com/t/nS56uH3xmt1j

**Comment by matanasov** [comment]
@theOrlin 
We still have this valid for Edge and IE11 even after the Column menu fixes (https://github.com/telerik/kendo-theme-default/issues/232). 

It is fine for other browsers. See this image:

![gridheader](https://user-images.githubusercontent.com/5861835/28175980-7623a1a2-67fe-11e7-8760-40daaa02ad2b.png)

**Comment by MilenaCh** [comment]
The same issue is valid for filter icon of TreeList:
https://www.screencast.com/t/oavumCJa4G

**Comment by 06b** [comment]
Applying the possible fix for #219 except making sure to only target ie & edge seems to fix the issue on the grid and the treelist... at least as a workaround

```
.k-edge .k-grid-header .k-header > .k-link, 
.k-ie .k-grid-header .k-header > .k-link {
    display: inline;
}
```
